op-batcher/metrics/metrics.go

```diff
@@ -326,8 +341,14 @@ func estimateBatchSize(block *types.Block) uint64 {
 		if tx.IsDepositTx() {
 			continue
 		}
-		// Add 2 for the overhead of encoding the tx bytes in a RLP list
-		size += tx.Size() + 2
+		bigSize := types.EstimatedL1SizeScaled(tx.RollupCostData())
+		bigSize = bigSize.Div(bigSize, big.NewInt(1e6)) // unscale it
```

Comment:
i don't love that we now have to sprinkle the `1e6` constant everywhere that `EstimatedL1SizeScaled` is used... can we upstream something to unscale it? it will break places like this if we want to change the scale in the future

Reply:
https://github.com/ethereum-optimism/op-geth/pull/424 ?

Reply:
updated as described with the upstream change.